configuration.make

```diff
@@ -30,7 +30,10 @@ ifeq (msys,$(findstring msys,$(lmi_build_type)))
 else ifeq (cygwin,$(findstring cygwin,$(lmi_build_type)))
   platform_makefile := msw_cygwin.make
 else
-  ifeq (mingw32,$(findstring mingw32,$(LMI_TRIPLET)))
+  kernel_release := $(shell uname -r)
+  ifeq (Microsoft,$(findstring Microsoft,$(kernel_release)))
+    platform_makefile := msw_wsl.make
```

Comment:
Do we really need a separate new `msw_wsl.make` file? Not sure what are the differences between it and the existing files exactly.

Reply:
`msw_generic.make` can't be used because of errors on commands execution:
```
w0 := $(shell winepath -w $(localbindir) | sed -e's/\\/\\\\/g')
```
`msw_cygwin.make` defines and uses `gcc_bin_dir`:
```
mingw_dir   := /opt/lmi/${LMI_COMPILER}_${LMI_TRIPLET}/gcc_msw
gcc_bin_dir := $(mingw_dir)/bin/
```
So a lot of command should be redefined:
```
AR      := $(gcc_bin_dir)$(host_hyphen)ar
CC      := $(gcc_bin_dir)$(host_hyphen)gcc
CPP     := $(gcc_bin_dir)$(host_hyphen)cpp
CXX     := $(gcc_bin_dir)$(host_hyphen)g++
LD      := $(gcc_bin_dir)$(host_hyphen)g++
RC      := $(gcc_bin_dir)$(host_hyphen)windres
```

Reply:
Hmm, not sure whether `msw_generic` makefile should use Wine-specific `winepath`. Maybe this should be defined elsewhere.

As for `gcc_bin_dir`, what exactly is the problem with defining it appropriately (and differently from Cygwin) under WSL?